docs/en/seatunnel-engine/resource-isolation.md

```diff
@@ -0,0 +1,83 @@
+---
+
+sidebar_position: 9
+-------------------
+
+After version 2.3.6. SeaTunnel can add `tag` to each worker node, when you submit job you can use `tag_filter` to filter the node you want run this job.
+
+# How to archive this:
+
+1. update the config in `hazelcast.yaml`,
+
+```yaml
+hazelcast:
+  cluster-name: seatunnel
+  network:
+    rest-api:
+      enabled: true
+      endpoint-groups:
+        CLUSTER_WRITE:
+          enabled: true
+        DATA:
+          enabled: true
+    join:
+      tcp-ip:
+        enabled: true
+        member-list:
+          - localhost
+    port:
+      auto-increment: false
+      port: 5801
+  properties:
+    hazelcast.invocation.max.retry.count: 20
+    hazelcast.tcp.join.port.try.count: 30
+    hazelcast.logging.type: log4j2
+    hazelcast.operation.generic.thread.count: 50
+  member-attributes:
+    group:
+      type: string
+      value: platform
+    team:
+      type: string
+      value: team1
+```
+
+In this config, we specify the tag by `member-attributes`, the node has `group=platform, team=team1` tags.
+
+2. add `tag_filter` to your job config
+
+```hacon
+env {
+  parallelism = 1
+  job.mode = "BATCH"
+  tag_filter {
+    group = "platform"
+    team = "team1"
+  }
```

Comment:
How to adapt to flink/spark？

env{} is visible to all engines

Reply:
At now, it is fine to only take effect under the zeta engine. We should not mention it on other engine's doc.

Reply:
ok